CHANGELOG.md

```diff
@@ -1,1346 +1,100 @@
 # Release 1.0.0
 
-## What's New
-
-* Bugfixes
-
-## DEFAULT Bind/Dial SERVICE PERMISSIONS FOR Admin IDENTITIES HAVE CHANGED
-
-Admin identities were able to Dial and Bind all services regardless of the effective service policies
-prior to this release. This could lead to a confusing situation where a tunneler that was assuming an Admin
-identity would put itself into an infinite connect-loop when a service's host.v1 address overlapped with
-any addresses in its intercept configuration.
-
-Please create service policies to grant Bind or Dial permissions to Admin identities as needed.
-
-## Component Updates and Bug Fixes
-
-* github.com/openziti/ziti: [v0.34.2 -> v1.0.0](https://github.com/openziti/ziti/compare/v0.34.2...v1.0.0)
-  * [Issue #1781](https://github.com/openziti/ziti/issues/1781) - Admin identities have bind and dial permissions to services
-
-# Release 0.34.2
-
-## What's New
-
-* The circuit id is now available in the SDK on the client and hosting side
-   * Requires 0.34.2+ routers
-   * Requests SDK support. Currently supported in the Go SDK 0.23.11+
-* Bug fixes
-
-## Component Updates and Bug Fixes
-* github.com/openziti/edge-api: [v0.26.13 -> v0.26.14](https://github.com/openziti/edge-api/compare/v0.26.13...v0.26.14)
-* github.com/openziti/sdk-golang: [v0.23.14 -> v0.23.15](https://github.com/openziti/sdk-golang/compare/v0.23.14...v0.23.15)
-* github.com/openziti/secretstream: [v0.1.17 -> v0.1.18](https://github.com/openziti/secretstream/compare/v0.1.17...v0.1.18)
-    * [Issue #24](https://github.com/openziti/secretstream/issues/24) - Potential side channel issue
-    * [Issue #25](https://github.com/openziti/secretstream/issues/25) - Reads from crypto/rand not checked for errors
-
-* github.com/openziti/ziti: [v0.34.1 -> v0.34.2](https://github.com/openziti/ziti/compare/v0.34.1...v0.34.2)
-    * [Issue #1831](https://github.com/openziti/ziti/issues/1831) - Circuit ID should be returned in the response to a Dial request
-    * [Issue #1873](https://github.com/openziti/ziti/issues/1873) - host.v1 health check time.Duration unconvertible
-
-# Release 0.34.1
-
-## What's New
-
-* Updates version of go to 1.22.x
-    * As usual when updating the go version, this is the only change in this release
+## About 1.0
 
-# Release 0.34.0
+What does marking OpenZiti as 1.0 mean?
 
-## What's New
+### Backwards Compatibility
+We've guaranteed API stability for SDK clients for years and worked hard to ensure that routers 
+and controllers would be backwards and forward compatible. However, we have had a variety of 
+management API changes and CLI changes. For post 1.0 releases we expect to make additions to the 
+APIs and CLI, but won't remove anything until it's been first marked as deprecated and then only
```

Comment:
Do we need a deprecation policy spellout and linked to?

Here is a quick one I came up with:


```
# OpenZiti Feature Deprecation Policy

## Overview
This policy outlines the approach for deprecating features within OpenZiti. Our goal is to minimize disruption for our users while ensuring that OpenZiti continues to evolve and improve.

## Policy Statement
- We provide a minimum of one version notice before a feature is deprecated.
- Notification of deprecation will be included in the release notes of a version preceding deprecation.
- Deprecations that break backwards compatability will be denoted by a major version change.

## Process
1. **Announcement**: Deprecation will be announced in:
   - Release notes
2. **Deprecation Period**: The feature will remain functional for at least the current major version cycle of the announcement.
3. **Support**: During the deprecation period, support will be provided as follows:
   - Documentation on alternatives or replacements.
   - Technical support for migration via Discourse
4. **Removal**: After the deprecation period, the feature will be removed in the next major release.

```

Reply:
That sounds reasonable. Where would we put that doc? Top level of the project? In the project readme?

Reply:
I would say so. Other than that, "Deprecation.md" is the only other place I would suggest.